user_file_manager.py

```diff
@@ -0,0 +1,18 @@
+import os
+from datetime import datetime
+
+
+class UserFileManager():
+    def __init__(self, history_dir):
+        self.history_dir = history_dir
```

Comment:
same for self.histroty_dir. 
Please, replace to output_dir

Reply:
Andrew and I prefer "history_dir". We want to leave it as is if it is not critical.